meson_options.txt

```diff
@@ -15,6 +15,7 @@ option('max_path_length'          ,type: 'integer' ,value: 1024, description: 'M
 # Feature toggles.
 option('define_main'                     ,type: 'boolean' ,value: true, description: 'Define main() in snitch -- disable to provide your own main() function.')
 option('with_exceptions'                 ,type: 'boolean' ,value: true, description: 'Use exceptions in snitch implementation -- will be forced OFF if exceptions are not available.')
+option('with_multithreading'             ,type: 'boolean', value: true, description: 'Multi-threading support and thread local variables -- disable if needed.')
```

Comment:
This also needs using in https://github.com/snitch-org/snitch/blob/main/snitch/meson.build, otherwise the option does nothing :)

Reply:
I didn't see that file, I thought meson just magically did the right thing 😄 
I added the option to `snitch/meson.build`, just imitating the other settings.